src/explore-education-statistics-frontend/src/modules/table-tool/components/TimePeriodDataTable.tsx

```diff
@@ -46,6 +46,20 @@ const TimePeriodDataTable = (props: Props) => {
       [options => options.length],
     );
 
+    // strips siblingless "Total" filters (columns/rows)
+    for (let i = 0; i < sortedFilters.length; i += 1) {
```

Comment:
Iterating over elements like this is not encouraged unless its performance critical code (don't think this is one of those instances). We can use the much more readable `Array.forEach`, and even that isn't necessarily recommended as it creates side-effects. Prefer to use `map` and `filter` instead.